sites/docs/src/content/docs/nf-core-tools/test-datasets/list.md

```diff
@@ -0,0 +1,44 @@
+---
+title: List test-datasets
+subtitle: List module/pipeline test files from the nf-core/test-datasets
+shortTitle: list
+weight: 10
+parentWeight: 30
+---
+
+The `nf-core test-datasets list` subcommand provides functionality to list existing test-datasets for pipelines and modules from the commandline.
+Test-datasets are hosted on the on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets/) github repository.
+
+:::note
+Not all files found on github can be listed with this subcommand.
+Some auxiliary files like `README`, or `LICENSE` as well as all files starting with `.` are always ignored.
+:::
+
+## Listing a file tree
+
+The following example lists the first entries of the file tree with test inputs for a pipeline (cut off after 25 lines of output).
+A branch name is always required to avoid a high number of requests against the github API.
+If no branch name is specified via the command line, the user will be prompted to enter one.
+
+<!-- RICH-CODEX
+working_dir: tmp/nf-core-nextbigthing
```

Comment:
this directory might not always be available. either don't specify a working_dir or generate it first.